devito/operator/operator.py

```diff
@@ -1388,6 +1389,23 @@ def _get_nbytes(self, i):
 
         return nbytes
 
+    @cached_property
+    def visible_devices(self):
+        device_vars = (
+            'CUDA_VISIBLE_DEVICES',
+            'ROCR_VISIBLE_DEVICES',
```

Comment:
ROCR or ROCM

?

Reply:
It's not currently an option, so I don't think that's a good idea. If the user accidentally set it then you may get unexpected, hard-to-debug behaviours since Devito will act as if devices have been set, but the ROCM runtime will not